### ISSUE TITLE
Implement API and system for handling notes and quotes

### ISSUE THREAD

**Issue by andmos** [andmos/ReadingList#101]
When reading a book, taking notes in a structured way is key.
Extend ReadingList with support for storing and retrieving notes. 
Something like:
```
/api/notes/random # Get a random note
/api/notes/random?author=harris,sam # Get a random note from Sam Harris books
```

The notes are markdown files on a give format for now, these needs to be stored and parsed in a sensible way.

**Comment by andmos** [comment]
Working on in #103

**Comment by andmos** [comment]
Closed in #103
